tests/superset_test_config.py

```diff
@@ -14,9 +14,10 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+# type: ignore
```

Comment:
Any reason for this change? It makes typing less strict.

Reply:
I'm getting tons of errors about variables from import * being overridden. Didn't find a way to bypass it and thought it wouldn't make much sense to `type: ignore` them one by one in a config file.